probot/server/tools/seed-db.js

```diff
@@ -19,9 +20,11 @@ db.then(() => {
     }
     if (names) {
       names.forEach(({ name }) => {
-        mongoose.connection.db
-          .dropCollection(name)
-          .catch(err => console.log(err));
+        if (name !== 'system.indexes') {
+          mongoose.connection.db
+            .dropCollection(name)
```

Comment:
I am not in favor of having drop the DB like this, you should instead drop only what you need. This is potentially dangerous, in situations where you would want other collections (for example, unrelated collections) on the DB to persist.

Instead of `for all` it would rather be great to have it `if only conditions.